src/applications/mhv/shared/hooks/usePrintTitle.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect } from 'react';
+
+const usePrintTitle = (
+  baseTitle,
+  userDetails,
+  dob,
+  dateFormat,
+  updatePageTitle,
+) => {
+  useEffect(
+    () => {
+      const { first, last, suffix } = userDetails;
+      const name = `${first} ${last} ${suffix}`
+        .replace(/\s*(undefined|null)\s*/g, '')
+        .trim();
```

Comment:
Just a suggestion, but you probably could get away with this and make it more readable instead of regex and `replace` `trim` manipulations. That is totally optional 
```suggestion
const { first = '', last = '', suffix = '' } = userDetails;
```